java/web/skeleton/functional/src/test/java/io/cecg/reference/Stepdefs.java

```diff
@@ -28,81 +28,26 @@ public class Stepdefs {
     private final String baseUri = SystemUtils.getEnvironmentVariable("SERVICE_ENDPOINT", "http://localhost:8080");
     private RequestSpecification request;
     private Response response;
-    private String unique_id;
```

Comment:
thinking out loud, should we have single nft and functional for both java and go applications?

Reply:
Why?

Reply:
if they are the same functionality, would avoid duplication. would also make sure that both applications stay consistent

Reply:
This code supposed to be used by the clients.
So if they want to write their app in java, most probably they would want to have tests also in java. The same for go